src/server/lib/renderer.tsx

```diff
@@ -133,14 +133,17 @@ export const renderer: <P extends RoutePaths>(
         <link rel="icon" href="https://static.guim.co.uk/images/${favicon}">
         <title>${pageTitle} | The Guardian</title>
         <script>window.gaUID = "${gaUID.id}"</script>
+
+        <script src="https://polyfill.io/v3/polyfill.min.js?features=Object.fromEntries" defer></script>
```

Comment:
wondering if we should use this URL or the assets.guim one used in the PR Tom linked: `https://assets.guim.co.uk/polyfill.io/v3/polyfill.min.js` https://github.com/guardian/frontend/pull/24473/files#diff-8b9cd752fdbccf643a1ce3fe63206eb1ebf9295f81bc915f6d7cb8fa6589e426R1

Reply:
I suppose `assets.guim.co.uk` is behind our own caching policies, so we can use that.